src/iris/api.py

```diff
@@ -6219,12 +6221,22 @@ class SenderHeartbeat():
     internal_allowlist_only = True
 
     def __init__(self, config):
-        cfg = config.get("iris-message-processor", {})
-        self.zk_hosts = cfg.get("zk_hosts", "127.0.0.1:2181")
+        cfg = config.get("external_sender", {})
         self.lock_path = cfg.get("lock_path", "/iris_message_processor/db_locks/bucket_lock")
         self.number_of_buckets = cfg.get("number_of_buckets", 100)
-        self.sender_ttl = config.get("sender_ttl", 60)
-        self.zk_debug = config.get("zk_debug", True)
+        self.sender_ttl = cfg.get("sender_ttl", 60)
```

Comment:
We moved sender_ttl and zk_debug from top level keys to keys under external_sender, which is a breaking change. Is that expected?